ProjectBank/Core/IUserRepository.cs

```diff
@@ -0,0 +1,14 @@
+namespace ProjectBank.Core;
+
+public interface IUserRepository
```

Comment:
Skal disse måske laves som Async?

Reply:
This is not possible for methods that don't have a body

Reply:
De laves Async ved at have en Task som return type